enable console prompts in settings

w-e-w opened the pull request:
## Description
add a UI settings under system for `--enable-console-prompts`
no reason why this should be restricted to a cmd arg
## Screenshots/videos:
![image](https://github.com/AUTOMATIC1111/stable-diffusion-webui/assets/40751091/df4deb0a-3236-4276-b85e-5d5cb5fd5b0c)


## Checklist:

- [x] I have read [contributing wiki page](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Contributing)
- [x] I have performed a self-review of my own code
- [x] My code follows the [style guidelines](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Contributing#code-style)
- [x] My code passes [tests](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Tests)

w-e-w committed:
enable console prompts in settings

AUTOMATIC1111 commented:
should probably then remove the commandline option (or make it do nothing) because if the commandline option is on, the setting stops working.

w-e-w committed:
deprecate --enable-console-prompts

use --enable-console-prompts as the default value for shared.opts.enable_console_prompts

w-e-w commented:
I made it so that `--enable-console-prompts` is the default value of `shared.opts.enable_console_prompts`
this way old users that are using `--enable-console-prompts` will have this setting set to True 
for normal users the default will be false

after they have applied / saved the setting the value in settings will take precedence and `--enable-console-prompts` ignored

I think it's a good way of transiting between the settings
but this also has the downside of my causing some confusion for users that are using `--enable-console-prompts` now and then decide to not use it later on

w-e-w committed:
lint
